src/o365/spfx/commands/project/project-upgrade.ts

```diff
@@ -374,6 +379,20 @@ class SpfxProjectUpgradeCommand extends BaseProjectCommand {
     cb();
   }
 
+  private writeReportTourFolder(findingsToReport: any, cmd: CommandInstance, options: Options): void {
+    const toursFolder: string = path.resolve(`${this.projectRootPath}/.tours`);
```

Comment:
To avoid issues with path separators, we should use `path.join` instead of `path.resolve`